package.json

```diff
@@ -15,7 +15,9 @@
     "clean": "rimraf lib",
     "prepublish": "npm run build",
     "watch": "tsc -w",
-    "test": "jest"
+    "test": "jest",
+    "lint": "tslint --project .",
+    "tslint-check": "tslint-config-prettier-check ./tslint.json"
```

Comment:
I don't see this target used anywhere?

Reply:
I used it locally. It tells you what tslint config options conflict with prettier.